WordPress/WordPressTest/System/3DTouch/WP3DTouchShortcutCreatorTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+import WordPress
+
+class WP3DTouchShortcutCreatorTests: XCTestCase {
+    var testShortcutCreator: WP3DTouchShortcutCreator!
+    
+    override func setUp() {
+        super.setUp()
+        testShortcutCreator = WP3DTouchShortcutCreator()
+    }
+    
+    override func tearDown() {
+        testShortcutCreator = nil
+        super.tearDown()
+    }
+    
+    func testCreateShortcutLoggedOutCreatesLoggedOutShortcuts() {
+        testShortcutCreator.createShortcuts(false)
+        XCTAssertEqual(UIApplication.sharedApplication().shortcutItems!, testShortcutCreator.loggedOutShortcutArray())
+    }
+}
```

Comment:
Logged In state has so many permutations (single site, multi-site, etc). I just didn't spend the time mocking out all the scenarios to make tests pass consistently.